ibm/mas_devops/roles/db2/defaults/main.yml

```diff
@@ -1,12 +1,17 @@
 ---
+
+#db2 action
+# ---------------------------------------------------------------------------
+db2_action: "{{ lookup('env', 'DB2_ACTION') | default('install', true) }}"
+
 # Configure Db2 instance
 # -----------------------------------------------------------------------------------------------------------------
 db2_namespace: "{{ lookup('env', 'DB2_NAMESPACE') | default('db2u', True) }}"
 db2_channel: "{{ lookup('env', 'DB2_CHANNEL') }}"
 
 db2_instance_name: "{{ lookup('env', 'DB2_INSTANCE_NAME') }}" # e.g. db2u-iot or db2u-manage
 db2_dbname: "{{ lookup('env', 'DB2_DBNAME') | default('BLUDB', true) }}"
-db2_version: "{{ lookup('env', 'DB2_VERSION') | default('s11.5.8.0', true) }}"
+db2_version: "{{ lookup('env', 'DB2_VERSION') }}"
```

Comment:
Was this intentional change to remove the default?  The default wasn't removed in the README?

Reply:
I've put it back aheead of merging this